src/modules/Convolution/info.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Convolution",
+    "description": "Image Convolution using a given 3x3 kernel matrix",
+    "inputs": {
+        "kernelValues": {
+            "type": "a string of space separated numbers",
```

Comment:
One small change, make this String, and add this part in the description.
```js
"type": "String",
"desc": "nine space separated numbers representing the kernel values in left to right and top to bottom format."
```

Reply:
Yes, we can have a constant factor as an input. It will be a different input field right? I will make the requested changes! Thanks!